### PR TITLE
use res.end: reverting #27

### PR DESCRIPTION
**Pull request by hzoo**
Revert's https://github.com/webpack/webpack-dev-middleware/pull/27

Assuming https://github.com/webpack/webpack-dev-middleware/issues/58 is correct, I'm just adding a PR to revert.

Otherwise feel free to ignore/close.

### CODE REVIEW

File: middleware.js [hunk 1]
```diff
@@ -186,7 +186,7 @@ module.exports = function(compiler, options) {
 					res.setHeader(name, options.headers[name]);
 				}
 			}
-			res.send(content);
+			res.end(content);
```
Comment (kaicataldo) [inline middleware.js]: What about a guard that defaults to the original implementation when Express isn't used (`karma-webpack` as an example) 
```
if (res.send) res.send(content);
else res.end(content);
```

Comment (hzoo) [inline middleware.js]: Yeah that could work as well

Comment (sokra) [inline middleware.js]: :+1:

Comment (hzoo) [inline middleware.js]: Updated with another commit (let me know if you need me to squash instead)

**Comment by sokra** [comment]
Thanks

**Comment by hzoo** [comment]
Thanks for merging and your work on webpack!

**Pull request merged**